[2023-03-29 13:09:06,014] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: SP_500_DATA_PIPELINE_v2.extract_data_task manual__2023-03-29T13:08:59.565257+00:00 [queued]>
[2023-03-29 13:09:06,066] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: SP_500_DATA_PIPELINE_v2.extract_data_task manual__2023-03-29T13:08:59.565257+00:00 [queued]>
[2023-03-29 13:09:06,068] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 13:09:06,070] {taskinstance.py:1239} INFO - Starting attempt 1 of 3
[2023-03-29 13:09:06,072] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 13:09:06,122] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): extract_data_task> on 2023-03-29 13:08:59.565257+00:00
[2023-03-29 13:09:06,134] {standard_task_runner.py:52} INFO - Started process 12552 to run task
[2023-03-29 13:09:06,150] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'SP_500_DATA_PIPELINE_v2', 'extract_data_task', 'manual__2023-03-29T13:08:59.565257+00:00', '--job-id', '101', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp7w54x0nj', '--error-file', '/tmp/tmptt5zzs3u']
[2023-03-29 13:09:06,153] {standard_task_runner.py:77} INFO - Job 101: Subtask extract_data_task
[2023-03-29 13:09:06,294] {logging_mixin.py:109} INFO - Running <TaskInstance: SP_500_DATA_PIPELINE_v2.extract_data_task manual__2023-03-29T13:08:59.565257+00:00 [running]> on host 5ebc180c7cc0
[2023-03-29 13:09:06,475] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=SP_500_DATA_PIPELINE_v2
AIRFLOW_CTX_TASK_ID=extract_data_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T13:08:59.565257+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-29T13:08:59.565257+00:00
[2023-03-29 13:09:10,356] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/pandas_datareader/tiingo.py:234: FutureWarning: In a future version of pandas all arguments of concat except for the argument 'objs' will be keyword-only
  return pd.concat(dfs, self._concat_axis)

[2023-03-29 13:09:10,407] {logging_mixin.py:109} INFO - ingestion from api completed
[2023-03-29 13:09:10,510] {logging_mixin.py:109} INFO - file has been saved at :sp_500_data.csv
[2023-03-29 13:09:10,514] {logging_mixin.py:109} INFO - sp_500_data.csv
[2023-03-29 13:09:10,519] {python.py:175} INFO - Done. Returned value was: None
[2023-03-29 13:09:10,563] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=SP_500_DATA_PIPELINE_v2, task_id=extract_data_task, execution_date=20230329T130859, start_date=20230329T130906, end_date=20230329T130910
[2023-03-29 13:09:10,722] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-03-29 13:09:11,051] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
